### PR TITLE
Ditch ugly singleton classes

### PR DESCRIPTION
**Pull request by Vendicated**
This ditches all singleton classes (Aliucord, PluginManager, ...) in favour of multiple named exports.

Motivation:
Using classes for this use is very ugly. If you now want to access any member, you require an instance. To get an instance, you need to make whatever you're making also a class and use dependency injection or similar to get the instance. Now you also need to deal with constantly typing `this`. Overall, it is just very ugly

Solution:
Multiple named exports. This has the same function as singletons, in that you can have any members, control access to them. However, this has the added benefit that
- you dont have to deal with `this`
- you don't have to deal with creating an instance via `new`, this module is itself responsible for doing whatever it needs
- you don't have to jiggle around instances via dependency injection. Want to access a member of any api? Just import it directly. Want the entire module as object? `import * as MyModule from "..."`

The Api is still the same, now you can just import any parts of the api from anywhere with no strings attached

Workflow before:
```ts
export MyClass {
    // some other module is responsible of creating me and passing me this dependency
   constructor(private dependency) {}

   doTheFunny() {
       this.dependency.stuff()
   }
}
```

workflow after
```ts
// hey look i can take care of everything myself and noone needs to pass me the dependency
import { stuff } from "dependency"

export function doTheFunny() { stuff() }
```

### CODE REVIEW

File: bootstrap/index.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@
         const bundlePath = aliucordDir + "/Aliucord.js.bundle";
         if (!AliuFS.exists(bundlePath)) await download("https://raw.githubusercontent.com/Aliucord/AliucordRN/builds/Aliucord.js.bundle", bundlePath);
 
-        (globalThis._globals ??= {}).aliucord = AliuHermes.run(bundlePath);
+        globalThis.aliucord = AliuHermes.run(bundlePath);
```
Comment (Vendicated) [inline bootstrap/index.ts:36]: this is another not mentioned change that would also have to be updated in the rollup plugin. Why was this ever called _globals.aliucord? It seems the purpose is to "hide" it, but being able to access this conveniently is important for debugging things on the console. And if anyone were to use this programatically (why would you care? Let them do it if they really want to), they would still do it regardless of the name. And if this is for "avoiding name clashes", i think it should be obvious that "aliucord" is less likely to clash with something else than "_globals"

File: src/Aliucord.ts [hunk 2]
```diff
@@ -14,34 +13,31 @@ interface SettingsSchema {
...
-    settings!: Settings<SettingsSchema>;
 
-    debugWS = new DebugWS();
-    reactDevTools = new ReactDevTools();
+export * as pluginManager from "./api/PluginManager";
+export const logger = new Logger("Aliucord");
+export let settings: Settings<SettingsSchema>;
 
-    pluginManager: PluginManager = new PluginManager(this);
+export let load = async function loadFn() {
```
Comment (TymanWasTaken) [inline src/Aliucord.ts:21]: why not just `export async function load()`?

File: src/Aliucord.ts [hunk 3]
```diff
@@ -14,34 +13,31 @@ interface SettingsSchema {
...
+    // Make sure this is only called once
+    //@ts-ignore
+    load = void 0;
```
Comment (TymanWasTaken) [inline src/Aliucord.ts:24]: just set a variable, you don't need to override the function with weird js fuckery

File: src/Aliucord.ts (hunk 2 above)
Comment (Vendicated) [reply src/Aliucord.ts:21]: because of the following line, you can reassign variables but not functions

**Review by Juby210 — APPROVED** [review]

**Pull request merged**